ClientTests/PhoneNumberFormatterTests.swift

```diff
@@ -0,0 +1,137 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Sorry, I made the same mistake on the other PR. I think Xcode stripped out the leading blanks during copy/paste and I failed to realize that.